### ISSUE TITLE
When searching the face watchlist, two WEB API requests are sent for one [Search], resulting in two duplicate results for each

### ISSUE THREAD

**Issue by Yoshitaka-Fukushima** [ORBNETSYS/i-PRO-Active-Guard-Public#110]
Test version: plugin v2.0.0 as of 29th Oct on XProtect 2024R1

Phenomenon
As tittle.
This does not happen in sample application.

![image](https://github.com/user-attachments/assets/9341b411-cf4a-4105-8b9f-f6566bfb56dd)

According to wire shark packets, following commands are sent twice.

GET /ai/system/info
GET /ai/v1.0/servers/info
POST /ai/v1.0/matchinglist/list 
GET /ai/v1.0/thumbnail?thumbnail-key

[Face watchlist.zip](https://github.com/user-attachments/files/17580830/Face.watchlist.zip)

@alexnaicuorbnet

**Comment by Yoshitaka-Fukushima** [comment]
@alexnaicuorbnet 
This is not fixed in v2.0.1 as of 1st Nov. 
This problem still occurs whether Management client is started or not.

**Comment by Yoshitaka-Fukushima** [comment]
@alexnaicuorbnet 
This is additional comment about the number of API requests. I hope this helps you.

POST /ai/v1.0/matchinglist/list
->Only this API is sent once as expected.

GET /ai/system/info
GET /ai/v1.0/servers/info
GET /ai/v1.0/thumbnail?thumbnail-key
->These API are sent multiple times.

**Comment by alexnaicuorbnet** [comment]
@Yoshitaka-Fukushima 
Thanks for the comment.

I solved this for 2.0.2.

I noticed that in the class SectionFaceGetResponseBody there are two fields for the matchinglist that get to be serialized.
At deserialization time these two matchinglists get to be merged into a single matchinglist with duplicated elements. 
My current fix is a workaround specific for this class, but you might want to change the class SectionFaceGetResponseBody, then the fix could be removed.

![image](https://github.com/user-attachments/assets/9e0ead20-eff2-4eac-9139-283956438d5e)

Another similar class SectionPeopleGetResponseBody doesn't have two serializable matchinglist fields:

![image](https://github.com/user-attachments/assets/abf06f6b-f0d4-4e07-8a99-ba7401092f8d)
